LibraBFT/Impl/OBM/ECP-LBFT-OBM-Diff/ECP-LBFT-OBM-Diff-1.agda

```diff
@@ -75,17 +136,47 @@ e_EpochManager_startNewEpoch-D em = fromEither ∘ e_EpochManager_startNewEpoch
 
 ------------------------------------------------------------------------------
 
-postulate -- TODO-1: e_EpochManager_checkEpc
- e_EpochManager_checkEpc
-  : EpochManager → EpochChangeProof
-  → Either ErrLog Unit
+e_EpochManager_checkEpc : EpochManager → EpochChangeProof → Either ErrLog Unit
```

Comment:
Eventually, before proving anything about this function, we'll want it in `EitherD`.  I'm OK with leaving that until then, but maybe add a TODO-1?

Reply:
OK